npm/translatePolicy.go

```diff
@@ -28,10 +28,11 @@ func craftPartialIptEntrySpecFromPort(portRule networkingv1.NetworkPolicyPort, s
 	}
 
 	if portRule.Port != nil {
+		portName := "port_" + portRule.Port.String()
 		partialSpec = append(
 			partialSpec,
 			sPortOrDPortFlag,
-			portRule.Port.String(),
+			portName,
```

Comment:
We can have the portRule without the Named port as well, right? @csfmomo/ @matmerr